README.md

```diff
@@ -1,101 +1,43 @@
-# Notion: the hassle-free node.js manager
-[![Build Status](https://travis-ci.org/notion-cli/notion.svg?branch=master)](https://travis-ci.org/notion-cli/notion)
-[![Build status](https://ci.appveyor.com/api/projects/status/2cohtlutserh8jfb/branch/master?svg=true)](https://ci.appveyor.com/project/stefanpenner/notion/branch/master)
+<p align="center">
+  <a href="https://www.notionjs.com/">
+    <img alt="Notion" src="https://github.com/notion-cli/notion/master/notion.png?raw=true" width="480">
```

Comment:
From messing around with URLs on GitHub, I think you actually need a `/blob/` before the branch (i.e. `/notion-cli/notion/blob/master`. As an example, look at these two URLs:

https://github.com/dherman/notion/readme/notion.png?raw=true

https://github.com/dherman/notion/blob/readme/notion.png?raw=true